### PR TITLE
CouchDBWallet

### PR DESCRIPTION
**Pull request by RohanShrothrium**
This PR implements CouchDBWallet store and fixes bugs in FileSystemWallet

### CODE REVIEW

File: scripts/check-env.sh [hunk 1]
```diff
@@ -37,6 +37,11 @@ else
    pip install tox
 fi
 
+echo "===> Installing couchdb"
+sudo apt update
```
Comment (dexhunter) [inline scripts/check-env.sh]: I don't think we can use `sudo` there.

Comment (RohanShrothrium) [reply scripts/check-env.sh]: I'm not sure how we can install `couchdb` on the azure instance.

Comment (yeasy) [reply scripts/check-env.sh]: We can pick the a popular linux, e.g., Ubuntu 18.04.

Comment (RohanShrothrium) [reply scripts/check-env.sh]: For the Azure VM @yeasy ? if so, it already uses `Ubuntu 18.04 image`.

**Pull request merged**